.travis.yml

```diff
@@ -16,8 +16,10 @@ env:
   # run tests/cpp
   - TASK=cpp_test
   # run tests/python
-  - TASK=python_test
```

Comment:
@mli asked me to remove Python and R tests. Please let me know if you want to keep it in or not.